run.go

```diff
@@ -98,9 +129,13 @@ func run(context *cli.Context) error {
 		}
 
 		// delete container's resources
-		if err := delete(containers[0].ID(), true); err != nil {
+		if err = delete(containers[0].ID(), true); err != nil {
 			return err
 		}
+
+		/* close and restore console */
+		console.Close()
```

Comment:
synchronization is missing there. We need to wait for for goroutine copying console to stdout so we don't miss any bites. That's usually done with a sync.WaitGroup (runc has that). You need to:

- Add a wg.Add(1) just before `go io.Copy(os.Stdout, console)`
- Add a wg.Done() in the go  routine copying console to stdout
- Add a wg.Wait() here.